test/variants/bool_unpack03.v3

```diff
@@ -0,0 +1,38 @@
+//@execute false=true; true=true
+def main(b: bool) -> bool {
```

Comment:
What I've generally done for tests like these is to use a return type of `int` and then return `-11`, `-12`, etc for each subsequent case. It makes easier to debug in case it should fail, because then you have a return value rather than working backward from a line number, which won't be available by default (without the runtime).